create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -195,13 +195,13 @@ void shouldLogExceptionMessageWhenExceptionOccurs() {
 
     @Test
     void shouldExtractOnlyScopusIdentifierIgnoreAllOtherIdentifiersAndStoreItInPublication() throws IOException {
-        var scopusIdentifiers = keepOnlyTheScopusIdentifiers();
+        var scopusIdentifiers = scopusData.getDocument().getMeta().getEid();
```

Comment:
Terje ønsket seg scopusIdentifier på format "2-s2.0-0027072373" istedet for "0027072373"